src/routes/forms.test.js

```diff
@@ -436,10 +434,6 @@ describe('Forms route', () => {
 
       expect(response.statusCode).toEqual(notFoundStatusCode)
       expect(response.headers['content-type']).toContain(jsonContentType)
-      expect(response.result).toMatchObject({
-        error: 'Not Found',
-        message: `Form with id '${id}' not found`
-      })
```

Comment:
Mind if we keep these lines?

It's important that we check the error message thrown by the form metadata methods is either passed all the way through OR replaced with something generic like this